### PR TITLE
Add support for investments product

### PR DESCRIPTION
**Pull request by maxdjohnson**

### CODE REVIEW

File: lib/plaid/products/investments.rb [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    # start_date   - Start of query for investment transactions.
+    # end_date     - End of query for investment transactions.
+    # account_ids  - Specific account ids to fetch transactions for (optional).
+    # count        - Amount of investment transactions to pull (optional).
+    # offset       - Offset to start pulling investment transactions (optional).
+    # options      - Additional options to merge into API request.
+    #
+    # Returns GetResponse.
+    def get(access_token, start_date, end_date,
+            account_ids: nil, count: nil, offset: nil, options: nil)
```
Comment (pbernasconi) [inline lib/plaid/products/investments.rb]: I don't really understand why we have `options` as a param in addition to `account_ids`, `count` and `offset`, but realize this is the same as our Tnx function - so all good

**Comment by pbernasconi** [comment]
@be9 can you take a pass at this too por favor?

File: lib/plaid/products/investments.rb (hunk 1 above)
Comment (be9) [reply lib/plaid/products/investments.rb]: It's been a recurring topic. For some reasons people want to merge in additional options.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 7.1.0 17-June-2019
```
Comment (be9) [inline CHANGELOG.md]: Is it planned to release on 6/17? To nitpick, it should be `17-Jun-2019`

Comment (maxdjohnson) [reply CHANGELOG.md]: yep. ⚡️

**Pull request merged**